### PR TITLE
create issue templates

### PR DESCRIPTION
**Pull request by pavlenex**
This PR creates issue templates.
The goal is to better organize issue submission, direct people to the proper channel to ask questions (Slack) or the proper place to submit ideas/feature requests (discussions).

Live preview https://github.com/pavlenex/Guide/issues/new/choose

Screenshot 2021-06-07 at 17 57 01

Could require some setup in the settings though https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-templates

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+blank_issues_enabled: false
+contact_links:
+  - name: 🚀 Discussions
+    url: https://github.com/btcpayserver/btcpayserver/discussions
+    about: Questions, suggestions and feature requests
+    about: Check our documentation for answers to common questions
```
Comment (pavlenex) [inline .github/ISSUE_TEMPLATE/config.yml:6]:
```suggestion
```

**Comment by johnsBeharry** [comment]
YOLO merge johns is here!

Just one thing to make clear. There should be a follow up PR with the rest of issue types as activating the templates deemphasises starting blank issues and could be a UX problem for people not as familiar with Github as it would seem like only bugs can be reported.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: 🚀 Discussions
+    url: https://github.com/btcpayserver/btcpayserver/discussions
```
Comment (johnsBeharry) [inline .github/ISSUE_TEMPLATE/config.yml:4]: gotta change this one.

Comment (pavlenex) [reply .github/ISSUE_TEMPLATE/config.yml:4]: hahaahah sorry!

**Comment by Bosch-0** [comment]
Would be good to have an 'image' or 'graphics' template. We have a lot of issues requesting images for various sections of the guides.

**Comment by GBKS** [comment]
Agree with @Bosch-0 on a template for image requests. How about the following templates?
- Content request
- Image request
- Feature request
- Discussions (Github discussions)
- Community chat (Slack link)
- Bug report

**Pull request closed without merging**